README.md

```diff
@@ -334,6 +342,36 @@ The `{%= path %}` value contains additional directory structure from the current
 
 NOTE: for grunt-responsive-images to pick up images within subdirectories you must set your files.src property to `**/*.{jpg,gif,png}`.
 
+#### Custom Input/Output options
+The options `customIn` and `customOut` pass additional arguments to the graphics engine.  Refer to your graphics engine 
+documentation for a full list of the possibilities.  `customIn` places arguments after the `convert` command and before
+any other arguments, while `customOut` places arguments after any other arguments, but before the output file name.
+
+```js
+grunt.initConfig({
+  responsive_images: {
+    myTask: {
+      options: {
+        customIn: ['-interlace', 'line'], // produce progressive images
+        customOut: [
+            // draw a copywrite notice in the bottom-right corner
+            '-gravity', 'SouthEast', '-font', "Arial", '-pointsize', '12',
```

Comment:
Consistent quote style might be good, so that it is not confusing in a quick look